apps/fxc-front/src/app/components/2d/path-element.ts

```diff
@@ -254,6 +250,63 @@ export class PathElement extends connect(store)(LitElement) {
     this.postScoreToHost(score);
   }
 
+  private computeScore(points: LatLon[]): Score {
+    const track: ScoringTrack = {
+      points: points.map((point, i) => {
+        return {
+          ...point,
+          alt: 0,
+          timeSec: i * 60,
+        };
+      }),
+      minTimeSec: new Date().getTime() / 1000,
```

Comment:
Why do we need `minTimeSec` - it could be inferred from the `timeSec` if needed.
Here we see that the values do not match

Reply:
`minTimeSec` is the instant when the track starts and `timeSec` is a time offset (time elapsed after minTimeSec).
The naming is not good. I renamed `minTimeSec` to `startTimeSec`
At the beginning, I wanted to be as close as possible with the existing data structures (RuntimeTrack in this case), but it turns out that it was not the best idea.